php/class-photonfill.php

```diff
@@ -235,7 +235,7 @@ public function set_original_src( $image, $attachment_id, $size, $icon ) {
 		 */
 		public function set_image_sizes() {
 			// Get theme set image sizes.
-			$this->image_sizes = apply_filters( 'photonfill_image_sizes', $this->image_sizes );
+			$this->image_sizes = (array) apply_filters( 'photonfill_image_sizes', $this->image_sizes );
```

Comment:
Likely, what we want is to verify that the return value is _already_ an array and if not overwrite it with an empty array since we're relying on not only the fact that it is an array but also the structure of that array:

```suggestion
			$this->image_sizes = apply_filters( 'photonfill_image_sizes', $this->image_sizes );
			
			if ( ! is_array( $this->image_sizes ) ) {
				$this->image_sizes = [];
			}
```

Reply:
1st - I don't see how this is any different, but with more, lines of, code?! 🤔 
2nd - This is not related to the issue we are fixing with this pull request. It's more like a safety check. The source of the bug was in the `create_image_sizes` method.

Reply:
Good point about the "structure of the array". Your original feedback made more sense now.